db/migrate/20230227075447_create_runs.rb

```diff
@@ -0,0 +1,20 @@
+class CreateRuns < ActiveRecord::Migration[7.0]
+  def change
+    create_table :runs do |t|
+      t.date :date
+      t.time :start_time
+      t.time :end_time
+      t.string :status
```

Comment:
What do you think about setting this to `enum`? Maybe this can be done later, though.